aws-lambda/src/main.rs

```diff
@@ -1,12 +1,7 @@
-use std::sync::Arc;
-
 use http::{to_request, to_response};
 use lambda_http::{run, service_fn, Body, Error, Response};
 use runtime::init_runtime;
-use tailcall::async_graphql_hyper::GraphQLRequest;
-use tailcall::blueprint::Blueprint;
-use tailcall::config::reader::ConfigReader;
-use tailcall::http::{handle_request, AppContext};
+use tailcall::builder::TailcallBuilder;
```

Comment:
```suggestion
use tailcall::TailcallBuilder;
```
Much nicer import to have everywhere.